core/assets/bundles/bundle_nl.properties

```diff
@@ -118,619 +134,654 @@ players = {0} Spelers online
 zone.impact0078.description = <insert description here>
 zone.crags.description = <insert description here>
-settings.language = Language
+settings.language = Taal
 settings.data = Game Data
-settings.reset = Reset to Defaults
-settings.rebind = Rebind
-settings.controls = Controls
-settings.game = Game
-settings.sound = Sound
-settings.graphics = Graphics
-settings.cleardata = Clear Game Data...
-settings.clear.confirm = Are you sure you want to clear this data?\nWhat is done cannot be undone!
-settings.clearall.confirm = [scarlet]WARNING![]\nThis will clear all data, including saves, maps, unlocks and keybinds.\nOnce you press 'ok' the game will wipe all data and automatically exit.
-paused = [accent]< Paused >
-clear = Clear
-banned = [scarlet]Banned
-yes = Yes
-no = No
-info.title = Info
-error.title = [crimson]An error has occured
-error.crashtitle = An error has occured
+settings.reset = Terug naar fabrieksinstellingen
+settings.rebind = Verander
+settings.resetKey = Reset
+settings.controls = Bediening
+settings.game = Spel
+settings.sound = Geluid
+settings.graphics = Grafisch
+settings.cleardata = Wis Game Data...
+settings.clear.confirm = Weet je zeker dat je deze data wilt verwijderen?\nDit is niet terug te draaien!
+settings.clearall.confirm = [scarlet]WAARSCHUWING![]\nDit verwijderd alle darta, inclusief saves, mappen, technologie en toetsen.\nAls je op doorgaat wist de game al je data en stopt ie automatisch.
+paused = [accent]< Gepauzeerd >
+clear = Wis
+banned = [scarlet]Verbannen
+yes = Ja
+no = Nee
+info.title = Informatie
+error.title = [crimson]Een fout heeft zich voorgedaan
+error.crashtitle = Een fout heeft zich voorgedaan
 blocks.input = Input
 blocks.output = Output
 blocks.booster = Booster
 block.unknown = [LIGHT_GRAY]???
-blocks.powercapacity = Power Capacity
-blocks.powershot = Power/Shot
+blocks.powercapacity = Stroom Capaciteit
+blocks.powershot = Stroom/Shot
 blocks.damage = Damage
-blocks.targetsair = Targets Air
-blocks.targetsground = Targets Ground
-blocks.itemsmoved = Move Speed
-blocks.launchtime = Time Between Launches
-blocks.shootrange = Range
-blocks.size = Size
-blocks.liquidcapacity = Liquid Capacity
-blocks.powerrange = Power Range
-blocks.powerconnections = Max Connections
-blocks.poweruse = Power Use
-blocks.powerdamage = Power/Damage
-blocks.itemcapacity = Item Capacity
-blocks.basepowergeneration = Base Power Generation
-blocks.productiontime = Production Time
-blocks.repairtime = Block Full Repair Time
-blocks.speedincrease = Speed Increase
-blocks.range = Range
-blocks.drilltier = Drillables
-blocks.drillspeed = Base Drill Speed
+blocks.targetsair = Beschiets Lucht
+blocks.targetsground = Beschiets Grond
+blocks.itemsmoved = Beweeg Snelheid
+blocks.launchtime = Tijd tussen lanceren
+blocks.shootrange = Afstand
```

Comment:
`Bereik`

Reply:
(Issue occurs multiple times in this document.)

Reply:
its indeed a better fit